### PR TITLE
feat: Release new PersonsModal

### LINKED ISSUE #11352: Persons Modal for Filters shows only one page of persons
## Bug description

As the title says

## How to reproduce

1. Open a funnel, and open the persons modal for a step
2. Only <100 persons will be listed
3. I think the endpoint is not returning the `next` param

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: please provide

### LINKED ISSUE #11480: styling on insight persons modal is off on chrome
## Bug description

scroll bars currently display on insight person modal, interfering with recordings. This used to render properly

image

*Please describe.*  
*If this affects the front-end, screenshots would be of great help.*  

## How to reproduce

1. on latest chrome build, open insight person modal. happ[ens regardless of zoom
2.
3.

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog, version/commit: please provide

### LINKED ISSUE #11659: Historical trends modal shows incorrect description
## Bug description

Historical trends get the conversion rate (16.67%) and display it as an event count. See here:

CVR_2

How can an event be performed 16.67 times??

## How to reproduce

1. Go to funnels
2. Switch to historical trends view
3. Click on the datapoint

## Environment

- [x] PostHog Cloud
- [x] self-hosted PostHog, version/commit: please provide

## Additional context

https://posthog.slack.com/archives/C03P1U69R6U/p1662381235606889

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

This is ready to go. In addition there are various other issues with the old Persons Modal, especially around session reocrdings popup so I think it's worth going for it.

Closes https://github.com/PostHog/posthog/issues/11659
Closes https://github.com/PostHog/posthog/issues/11480
Closes https://github.com/PostHog/posthog/issues/11352

## Changes

* Removes old PersonsModal code and replaces it with the new stuff

### CODE REVIEW

**Comment by neilkakkar** [comment]
Hey @benjackwhite , just a heads up -> can't tell if this made it into the new persons modal, but we really want this for person on events!

https://github.com/PostHog/posthog/pull/11500

**Comment by benjackwhite** [comment]
@neilkakkar this should have got merged in properly. Our team is flagged for the new modal if you want to try it?

**Comment by benjackwhite** [comment]
@neilkakkar looks like I missed something! I'll get it fixed 👍

**Comment by neilkakkar** [comment]
Hey @benjackwhite , tried it!

image

It seems to not work with pagination, like it does in the old modal? (example: The count here should be atleast 7). Every 'load more' action refreshes the missing_persons count, vs. adding it to any existing count, like we do with the number of persons.

**Comment by benjackwhite** [comment]
> It seems to not work with pagination, like it does in the old modal? (example: The count here should be atleast 7). Every 'load more' action refreshes the missing_persons count, vs. adding it to any existing count, like we do with the number of persons.

Right that makes sense, it was tricky to get the diff right when I merged so I think I missed a crucial step 😅  - should be fixed here https://github.com/PostHog/posthog/pull/11694/commits/e32b25a0c0d5b260d11a4e4c5f61e0192e63eb4a

File: frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.test.ts [hunk 1]
```diff
@@ -145,8 +145,8 @@ describe('sessionRecordingDataLogic', () => {
                 .toDispatchActions(['loadRecordingMeta', 'loadRecordingMetaSuccess', 'loadEvents'])
                 .toMatchValues({
                     eventsApiParams: {
-                        after: '2021-12-09T19:35:59+00:00',
-                        before: '2021-12-09T20:23:24+00:00',
+                        after: '2021-12-09T19:35:59Z',
+                        before: '2021-12-09T20:23:24Z',
```
Comment (benjackwhite) [inline frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.test.ts:149]: @mariusandra I had to make this change here. I spent a good hour looking into why but I couldn't figure it out. It seems like a pretty pure code path calling `dayjs.utc(TIME).format()` where `TIME` hasn't changed in this PR (I checked).

Feels like somewhere dayjs is being mutated in a way that this PR has modified but I can't figure out why. Perhaps you have some ideas?

**Review by mariusandra — APPROVED** [review]
This is good for me. I'm really not sure what's happening with the timezone in that test, but we can fix it next time it comes up.

**Pull request merged**